data/scenarios/Challenges/bucket-brigade.yaml

```
@@ -0,0 +1,155 @@
+version: 1
+name: Bucket Brigade
+author: Karl Ostmo
+description: |
+  Convey resources with limited mobility
+creative: false
+robots:
+  - name: base
+    display:
+      char: 'Ω'
+      attr: robot
+    heavy: true
+    dir: [0, 1]
+    devices:
+      - scanner
+      - 3D printer
+      - dictionary
+      - branch predictor
+      - strange loop
+      - clock
+      - string
+      - ADT calculator
+      - comparator
+      - workbench
+      - grabber
+      - lambda
+      - hearing aid
+      - logger
+      - net
+    inventory:
+      - [10, repro kit]
+      - [1, treads]
+  - name: minerobot
+    system: true
+objectives:
+  - goal:
+      - Deliver some "coal" to the base.
+      - |
+        To excavate "coal" from the "mine" (M), a robot needs to
```

```suggestion
        To excavate "coal" from the "lignite mine" (M), a robot needs to
```

If you're going to put it in quotes it needs to match the name of the entity.  One of the bugs with my solution at first was that I was looking for a `mine` instead of a `lignite mine`, and I eventually had to send a robot to `scan` it just so I could figure out what it was called.

---

Agreed.  It would be cool if we could have some designated markup for entity names that we can typecheck/validate against the list of known entities for that scenario (c.f. #545).